### PR TITLE
Ansible 2.9: fixing deprecation warnings

### PR DESCRIPTION
**Pull request by barpavel**
Fixing deprecation warnings produced by tox & Ansible due to upgrading to Ansible 2.9:
1) ovirt_vm_facts --> ovirt_vm_info
2) ovirt_host_facts -->ovirt_host_info
3) ovirt_disk_facts --> ovirt_disk_info
4) ovirt_storage_vm_facts --> ovirt_storage_vm_info
5) ovirt_storage_template_facts --> ovirt_storage_template_info
6) ovirt_storage_domain_facts --> ovirt_storage_domain_info

Requirements documentation:
1) Updating the requirements to use Ansible 2.9.
2) Other improvements & fixes.

Signed-off-by: Pavel Bar <pbar@redhat.com>

### CODE REVIEW

**Review by mwperina — APPROVED** [review]
+1

**Review by mnecas — CHANGES_REQUESTED** [review]
You are missing 
```
tasks/clean/remove_invalid_filtered_master_domains.yml:      ovirt_storage_domain_facts:
tasks/clean/remove_valid_filtered_master_domains.yml:      ovirt_storage_domain_facts:
tasks/clean/update_ovf_store.yml:      ovirt_storage_domain_facts:
tasks/clean_engine.yml:      ovirt_storage_domain_facts:
tasks/clean_engine.yml:      ovirt_disk_facts:
tasks/recover/register_templates.yml:      ovirt_storage_template_facts:
tasks/recover/register_vms.yml:      ovirt_storage_vm_facts:
tasks/recover_engine.yml:      ovirt_storage_domain_facts:
```

**Comment by barpavel** [comment]
> You are missing
> 
> ```
> tasks/clean/remove_invalid_filtered_master_domains.yml:      ovirt_storage_domain_facts:
> tasks/clean/remove_valid_filtered_master_domains.yml:      ovirt_storage_domain_facts:
> tasks/clean/update_ovf_store.yml:      ovirt_storage_domain_facts:
> tasks/clean_engine.yml:      ovirt_storage_domain_facts:
> tasks/clean_engine.yml:      ovirt_disk_facts:
> tasks/recover/register_templates.yml:      ovirt_storage_template_facts:
> tasks/recover/register_vms.yml:      ovirt_storage_vm_facts:
> tasks/recover_engine.yml:      ovirt_storage_domain_facts:
> ```

**Comment by barpavel** [comment]
> > You are missing
> > ```
> > tasks/clean/remove_invalid_filtered_master_domains.yml:      ovirt_storage_domain_facts:
> > tasks/clean/remove_valid_filtered_master_domains.yml:      ovirt_storage_domain_facts:
> > tasks/clean/update_ovf_store.yml:      ovirt_storage_domain_facts:
> > tasks/clean_engine.yml:      ovirt_storage_domain_facts:
> > tasks/clean_engine.yml:      ovirt_disk_facts:
> > tasks/recover/register_templates.yml:      ovirt_storage_template_facts:
> > tasks/recover/register_vms.yml:      ovirt_storage_vm_facts:
> > tasks/recover_engine.yml:      ovirt_storage_domain_facts:
> > ```

I know, that's why this patch is still in "Draft" and its description has empty 3, 4 & 5 sections :) Actually it should be even 6 sections!
I will add the missing commits (after or during the long holiday weekend that we have in Israel) and will "request" for your review :)

File: README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ The `oVirt.disaster-recovery` role responsible to manage the disaster recovery s
 Requirements
 ------------
 
- * Ansible version 2.7.2 or higher
- * Python SDK version 4.2.4 or higher
+ * Ansible version 2.9 or higher
+ * Python SDK version 4.3 or higher
```
Comment (mwperina) [inline README.md:10]: Master development should aim only to 4.4

Comment (barpavel) [reply README.md]: Thanks, fixed :)

**Review by mwperina — APPROVED** [review]
+1

**Review by mnecas — APPROVED** [review]
LGTM

File: meta/main.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ galaxy_info:
   description: "Disaster recovery for oVirt engine"
   company: Red Hat, Inc.
   license: Apache License 2.0
-  min_ansible_version: 2.7.2
+  min_ansible_version: 2.9
```
Comment (shenitzky) [inline meta/main.yml:7]: We should also require 2.9 version on the spec file - 
ovirt-ansible-disaster-recovery.spec.in

Comment (barpavel) [reply meta/main.yml:7]: Done, thanks!

**Review by shenitzky — APPROVED** [review]

**Pull request merged**